meta/template/basic.lua

```diff
@@ -126,6 +126,12 @@ function loadfile(filename, mode, env) end
 ---@nodiscard
 function loadstring(text, chunkname) end
 
+---@version 5.1
+---@param proxy   boolean | table
```

Comment:
Please refer to the code style in this file, remove the spaces on the side of the `|`